app/design/frontend/base/default/template/algoliasearch/internals/configuration.phtml

```diff
@@ -257,6 +257,9 @@ $algoliaJsConfig = array(
 
 <script>
     window.algoliaConfig = <?php echo json_encode($algoliaJsConfig); ?>;
+    if (typeof algoliaHookAfterConfigLoad == 'function') {
+        window.algoliaConfig = algoliaHookAfterConfigLoad(algoliaConfig);
```

Comment:
Right now, we don't have a backend hook to modify generated configuration. So there'd be only one place to modify it - frontend.
Yes, having those hooks in general makes it harder to debug customers' issues, but from my experience those customers who are using hooks know what they are doing and do not have that kind of issues.